### ISSUE TITLE
Is this somewhat better than existing sam-washington/requests-aws4auth?

### ISSUE THREAD

**Issue by 1oglop1** [DavidMuller/aws-requests-auth#42]
Hi I just found out about this project which seems to be more recent, however there is this existing project https://github.com/sam-washington/requests-aws4auth - which suppose to do the same and is also used by elasticsearch-curator. 
Could you please tell me if there are any differences or improvements over mentioned project which also seems not to be maintained?

**Comment by DavidMuller** [comment]
Yes as you mentioned [`elasticsearch-py` currently recommends `requests-aws4auth`](https://elasticsearch-py.readthedocs.io/en/master/#running-on-aws-with-iam). 

> Could you please tell me if there are any differences or improvements over mentioned project which also seems not to be maintained?

Both libraries seek to sign your requests to AWS, so feel free to use whichever one fits most optimally in your workflows. It looks like [`requests-aws4auth`](
https://github.com/sam-washington/requests-aws4auth) has some more involved automatic key regeneration procedures that you could choose to opt into. Mostly depends on your use case -- my guess, however, is that you wouldn't have much trouble switching between the two projects later if you find one does/doesn't work for you.
